lava/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "lava-testnet-1",
+    "assets": [
+      {
+        "description": "The native token of Lava",
+        "denom_units": [
+          {
+            "denom": "ulava",
+            "exponent": 0
+          },
+          {
+            "denom": "lava",
+            "exponent": 6
+          }
+        ],
+        "base": "ulava",
+        "name": "LAVA",
+        "display": "lava",
+        "symbol": "LAVA",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lava/images/lava-icon.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lava/images/lava-icon.svg"
+        },
+        "coingecko_id": "lava",
```

Comment:
I don't think this id is used by CoinGecko for lava... It seems to be for lavaswap

Reply:
Corrected: 
[6bd2433](https://github.com/cosmos/chain-registry/pull/2244/commits/6bd24332679b38d296fc07f4a9ddb3693f139f2b)